### PR TITLE
store: update API in messages

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

leftover after the last API rename in #15811

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]
the naming seems asymmetric.
upload_to_cloud
but
load_model

**Comment by Borda** [comment]
> the naming seems asymmetric. `upload_to_cloud` but `load_model`

renamed to `upload_model`

File: src/lightning/store/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-from lightning.store.cloud_api import download_from_cloud, load_model, upload_to_cloud
+from lightning.store.cloud_api import download_from_cloud, load_model, upload_model
 
-__all__ = ["download_from_cloud", "load_model", "upload_to_cloud"]
+__all__ = ["download_from_cloud", "load_model", "upload_model"]
```
Comment (akihironitta) [inline src/lightning/store/__init__.py:3]: My comment here overlaps with Adrian's above comment but should it be named like this?
```suggestion
__all__ = ["download_model", "load_model", "upload_model"]
```

**Review by akihironitta — APPROVED** [review]

File: src/lightning/store/__init__.py (hunk 1 above)
Comment (Borda) [reply src/lightning/store/__init__.py]: done :)

**Pull request merged**